openquakeplatform/openquakeplatform/ript/templates/ript/tabs/fragility.html

```diff
@@ -0,0 +1,48 @@
+    <div class="alert alert-info" role="alert">1. Use this form to initialize the fragility model.</div>
+    <div id="form" class="menuItems">
+      <div>
+        <label>Fragility Model Id:</label>
+        <input id="functionId" type="text">
+      </div>
+      <div>
+        <label>Asset Category:</label>
+        <input id="assetCategory" type="text" placeholder="buildings">
+      </div>
+      <div>
+        <label>Loss Category:</label>
+        <select id="lossCategory">
+          <option value="structural">Structural</option>
+          <option value="nonstructural">Non-Structural</option>
+          <option value="contents">Contents</option>
+          <option value="business_interruption">Business Interruption</option>
+          <option value="occupants">Occupants</option>
+        </select>
+      </div>
+      <div>
+        <label>Description:</label>
+        <textarea id="functionDescription" rows="4" cols="10"></textarea>
+      </div>
+      <div>
+        <label>Damage States</label><br>
+        <label>(Comma Separated):</label>
+        <textarea id="limitStates" rows="4" cols="10">slight, moderate, collapse</textarea>
+      </div>
+    </div>
+    <br>
+    <div class="alert alert-info" role="alert">2. One can use the 'Add Discrete function' &amp; 'Add Continuous Function' buttons to generate as many discrete and continuous fragility functions sets (ffs) as needed.<br>
```

Comment:
I suggest changing this (and similar occurrences) to something like:
`Use the 'Add Discrete Function' and 'Add Continuous Function buttons to generate as many discrete and continuous fragility function sets (ffs) as needed.<br>
Generate a NRML file by clicking 'Convert to NRML' once tables are populated.`